js-src/compat/plugins/mondialrelay/checkout-mondialrelay.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Checkout scripts for: Mondial Relay - WordPress (by Kasutan).
+ */
+
+(function (root, factory) {
+	if ( typeof define === 'function' && define.amd ) {
+		define([], factory(root));
+	} else if ( typeof exports === 'object' ) {
+		module.exports = factory(root);
+	} else {
+		root.CheckoutMondialRelay = factory(root);
+	}
+})(typeof global !== 'undefined' ? global : this.window || this.global, function (root) {
+
+	'use strict';
+
+	var $ = jQuery;
+	var _hasJQuery = ( $ != null );
+
+	var _hasInitialized = false;
+	var _publicMethods = {};
+	var _settings = {
+		buttonSelector: '#delivery_point_chosen',
+	};
+
+
+
+	/**
+	 * METHODS
+	 */
+
+
+
+	/**
+	 * Update checkout fields.
+	 */
+	var updateCheckout = function() {
+		// Create update checkout event
+		var event = new Event( 'update_checkout' );
+
+		// Trigger event
+		document.body.dispatchEvent( event );
```

Comment:
- Since the event `update_checkout` is a jQuery event from the WooCommerce scripts, it would be appropriate to use the jQuery method here.

Reply:
@vlad-dedikov, you missed making this change.